standalone/pruntime/enclave/src/lib.rs

```diff
@@ -1600,3 +1600,30 @@ fn query(q: types::SignedQuery) -> Result<Value, Value> {
 fn test(param: TestReq) -> Result<Value, Value> {
     Ok(json!({}))
 }
+
+mod identity {
+    use super::*;
+    type WorkerPublicKey = sp_core::ecdsa::Public;
+
+    pub fn is_gatekeeper() -> bool {
+        let mut local_state = LOCAL_STATE.lock().unwrap();
```

Comment:
Lock the global state in a leaf function like this one is error-prone. It will be easy to encounter dead locks. It would be better to pass the storage in via parameter.

Reply:
fixed